packages/button/styles/variant-ghost.css

```diff
@@ -0,0 +1,63 @@
+.g-btn.variant-ghost {
+  color: var(--gray-3);
+  border-color: transparent;
+  background-color: transparent;
+
+  & .icon svg {
+    & [fill] {
+      fill: var(--gray-3);
+    }
+    & [stroke] {
+      stroke: var(--gray-3);
+    }
+  }
+
+  &.size-small,
+  &.size-medium {
+    padding: 0;
+  }
+
+  &:hover:not([disabled]) {
+    border-color: transparent;
+  }
+
+  &[disabled] {
+    &.background-light,
+    &.background-brand-secondary {
+      color: rgba(114, 114, 116, 0.3);
+      & .icon svg {
+        & [fill] {
+          fill: rgba(114, 114, 116, 0.3);
+        }
+        & [stroke] {
+          stroke: rgba(114, 114, 116, 0.3);
+        }
+      }
+    }
+    &.background-dark,
+    &.background-brand {
+      color: rgba(255, 255, 255, 0.3);
+      & .icon svg {
+        & [fill] {
+          fill: rgba(255, 255, 255, 0.3);
+        }
+        & [stroke] {
+          stroke: rgba(255, 255, 255, 0.3);
+        }
+      }
```

Comment:
- [ ] Going to check on this

Reply:
I think it makes sense to keep this the same as `tertiary-neutral`

Reply:
+1, I think the contrast between the `disabled` state is good for both variants